documentation/source/reference/molecule/atomtype.rst

```diff
@@ -116,3 +116,8 @@ Atom type       Description
 ``S6tt``        sulfur atom with no lone pairs (valance 6) and two triple bonds
 ``S6tdc``       charged sulfur atom with no lone pairs (valance 6), one to two triple bonds, up to two double bonds, and up to four single bonds
 =============== ==============================================================================================================================================================
```

Comment:
You need to delete line 118 for the table to format properly. Otherwise, it tries to end the table early